### PR TITLE
Polymorphism(Java) 

### LINKED ISSUE #1813: Polymorphism(Method Overloading && Method Overridding in Java)
Hello Mam(@SakshiShreya) I found examples of Inheritance in Java and C but didn't find examples/codes for Polymorphism in Java.Therefore I raised this issue.I would like to contribute for this issue using JAVA.It would great if you assign me this task.

### PR DESCRIPTION
**Pull request by CSElonewolf**
closes #1813 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:

1. Added Method Overriding example 
2. Added Method Overloading example
3. Added Constructor Overloading example
4. Added README.md

#### Languages Used:
Java

#### Files Added:

1. Added Method Overriding example 
2. Added Method Overloading example
3. Added Constructor Overloading example
4. Added README.md

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Polymorphism(Java)/Constructor_Overloading.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+public class Employee
+{
+//Constructor 1
```
Comment (jainaman224) [inline Polymorphism(Java)/Constructor_Overloading.java]: Incorrect indentation.

File: Polymorphism(Java)/Constructor_Overloading.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+public class Employee
```
Comment (jainaman224) [inline Polymorphism(Java)/Constructor_Overloading.java]: Incorrect folder name

File: Polymorphism(Java)/Constructor_Overloading.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+     Employee emp3 = new Employee("HCL", 12234);//calls constructor 3
+ }
+}
+
+/* Output:
+          Employee Details:
+          Employee name: Deep
+          Company name: HCL
+          Employee id: 12234
+          */
```
Comment (jainaman224) [inline Polymorphism(Java)/Constructor_Overloading.java]: Please remove constructor overloading.
Method overloading will serve the purpose.

File: Polymorphism(Java)/Method_Overriding.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
+//Super-class or Parent Class
+public class MainRepo {
+   private String name;
```
Comment (jainaman224) [inline Polymorphism(Java)/Method_Overriding.java]: Incorrect indentation.

**Comment by CSElonewolf** [comment]
@jainaman224  sir  I have updated the indentation.Please Review The changes and please do let me know if there is improvement that is needed to be made.

File: Polymorphism(Java)/Method_Overloading.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        System.out.println("Got Integer data.");
+    }
+    // this method accepts String object
+    private static void display(String a)
+    {
+        System.out.println("Got String object.");
+    }
+    //this method accepts float value
+     private static void display(float a)
```
Comment (jainaman224) [inline Polymorphism(Java)/Method_Overloading.java]: Incorrect indentation.
Please self the PR once.

Comment (CSElonewolf) [reply Polymorphism(Java)/Method_Overloading.java]: Like I am sorry but I can't figure out what is 4 space indentation?

Comment (SakshiShreya) [reply Polymorphism(Java)/Method_Overloading.java]: In line 14, there are 5 spaces.

**Comment by somya-kapoor** [comment]
@CSElonewolf Are you working on it?

**Comment by somya-kapoor** [comment]
@CSElonewolf Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**